packages/cli/generators/app/templates/src/controllers/index.ts.ejs

```diff
@@ -0,0 +1 @@
+export * from './ping.controller';
```

Comment:
This makes me wonder whether it should be included when the app is generated as opposed to adding it on when the controller generator is run. Thoughts?

Reply:
Check the file -- It's in the `app` generator and this is just the default `index.ts` that gets generated since we are generating `ping.controller` in new apps.

Reply:
The ping controller I know about, but I was wondering about whether we should include this `index.ts` file when the app generator is run